src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -187,10 +187,26 @@ private IList<AnnotatedCodeLocation> NormalizeRawMessage(string rawMessage, out
                     tokens = location.Split(':');
 
                     relatedLocations = relatedLocations ?? new List<AnnotatedCodeLocation>();
+                    PhysicalLocation physicalLocation;
 
-                    var relatedLocation = new AnnotatedCodeLocation
+                    if (tokens[0].Equals("file", StringComparison.OrdinalIgnoreCase))
+                    {
+                        // special case for file paths
```

Comment:
Consider naming the first one `csvTokens`. For symmetry. It feels wrong to have one generic name and one specific name.

---
In reply to: [134357737](https://github.com/Microsoft/sarif-sdk/pull/591#discussion_r134357737) [](ancestors = 134357737,134355305)